documentation/operators/src/archive/nodes/gateway-setup.md

```diff
@@ -1,22 +1,26 @@
 # Gateways
 
-> The Nym gateway was built in the [building nym](../binaries/building-nym.md) section. If you haven't yet built Nym and want to run the code, go there first.
+```admonish warning
+**This is an archived page for backwards compatibility. The content of this page is not updated since April 19th 2024. Eventually this page will be terminated!**
```

Comment:
I would say here that we're leaving this for backwards compatibility _for existing operators_ and that people wanting to set up new nodes should set up a `nym-node`

Reply:
And probably put some of it in bold to be sure